schedule/yast/remote_vnc/remote_vnc_controller_opensuse.yaml

```diff
@@ -20,4 +20,6 @@ schedule:
  - installation/start_install
  - installation/await_install
  - installation/reboot_after_installation
- - support_server/wait_children
+ - x11/validate_vnc_from_client
+test_data:
+  target_ip: "10.0.2.15"
```

Comment:
in openSUSE looks different, during install is 10.0.2.15 and after install there is new lease 10.0.2.16, if that is constant, we are fine, we can just edit this value, it is different behavior, which we could check with YaST team why it happens, if it is not constant among runs, you previous method to calculate the ip would be useful then.

Reply:
I cannot run VRs for opensuse as there is a bug. I will check if I can find a workaround but probably will create a separate ticket.